openstack/cinder/templates/vcenter_datacenter_cinder_deployment.yaml

```diff
@@ -49,6 +49,8 @@ template: |
         hostname: cinder-volume-vmware-{= name =}
         {{- tuple . "{= availability_zone =}" | include "utils.kubernetes_pod_az_affinity" | nindent 8 }}
         {{- include "utils.proxysql.pod_settings" . | nindent 8 }}
+        initContainers:
+        {{- tuple . (dict "service" (print "cinder-api," .Release.Name "-mariadb," .Release.Name "-rabbitmq")) | include "utils.snippets.kubernetes_entrypoint_init_container" | indent 8 }}
```

Comment:
Now that we have established, that `cinder-volume` talks directly to the DB, why do we still wait for `cinder-api`? This makes us effectively wait for the migration job, doesn't it?

Reply:
Yes, probably it doesn't make sense that way around, since `cinder-api` calls the `cinder-volume` and not the other way around.

> This makes us effectively wait for the migration job, doesn't it?

Yes, but not a specific migration job with a name that changes on every attribute change, but just a migration job. Either way, it is gone now.